service/controller/v20/resource/deployment/current.go

```diff
@@ -26,7 +26,7 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	var currentDeployments []*v1beta1.Deployment
 	{
 		namespace := key.ClusterNamespace(customResource)
-		deploymentList, err := r.k8sClient.Extensions().Deployments(namespace).List(apismetav1.ListOptions{})
+		deploymentList, err := r.k8sClient.ExtensionsV1beta1().Deployments(namespace).List(apismetav1.ListOptions{})
```

Comment:
You mean `AppsV1` - yeah, it's been available since k8s 1.9 (see https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md#apps) so it's safe to switch. Will revisit all affected update PRs.

Reply:
Oh wow yes I did mean `AppsV1`. :smile: Thanks and yes if its not too big a change it will help us get ready for 1.16.

I try to update it when I see it in charts as well.

Reply:
Fixed for this and other operators.